### DISCUSSION TITLE
availibility of static gridpoint data

### DISCUSSION THREAD

**Discussion by willsalz** [weather-gov/api#244] [Q&A]
Hey team, first off, thanks for all the work you do!

My question is: is the gridpoint geospatial data [ i.e. the gridpoints and their associated polygon geometry ] available for bulk download? It would allows us to do `(lat,lng) -> gridpoint` lookups offline and avoid a query to your API, reducing our load on your servers. Cheers.

**Comment by StephenClouse** [comment]
This is something we considered, but the surprise we found out is that these can change without notice. This is actually a defect we still need to fix, as it's something we didn't account for. Fixing this is another issue we plan to address in the near future (part of the backend service replacement I mentioned in #71).
